assets/js/blocks/cart-checkout/checkout/empty-cart.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { SHOP_URL } from '@woocommerce/block-settings';
+import { Icon, cart } from '@woocommerce/icons';
+
+const EmptyCart = () => {
+	return (
+		<div className="wc-block-checkout-empty">
+			<Icon
+				className="wc-block-checkout-empty__image"
+				alt={ __( 'Empty cart', 'woo-gutenberg-products-block' ) }
+				srcElement={ cart }
```

Comment:
Aha ! I see there is an empty cart icon intended for the "empty checkout" state, this might work for empty cart too.